src/Pirouette/Term/ToTla.hs

```diff
@@ -34,6 +34,7 @@ import           Control.Monad.Reader
 import           Control.Monad.Except
 import           Control.Monad.State
 import           Control.Arrow (first, second, (***))
+import           Control.Exception ( catch )
```

Comment:
Is this really needed? you added the import but didn't use the imported function in the change to `mkTLATySpecializer`

```suggestion
```

Reply:
Yes, you are right, I added it and then changed my mind and did not use it. It should be suppressed.